src/Articles.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import { getArticles } from "./utils/getAllArticles";
+import { Link } from "react-router-dom";
+import "./styles/articles.css";
+
+const Article = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getArticles()
+      .then((data) => {
+        setArticles(data);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
It's great we have a catch block and a console for us devs, but our users need to see if there's an error. How could we handle this?